lib/tasks/support.rake

```diff
@@ -91,6 +91,16 @@ namespace :support do
     end
   end
 
+  desc "Unsubscribe all Brexit checker subscriptions"
+  task unsubscribe_all_brexit_checker_subscriptions: :environment do
+    unsubscribe_time = Time.zone.now
+
+    brexit_subscriptions = Subscription.joins(:subscriber_list).where("ended_at IS NULL").where("subscriber_lists.tags::text like '%brexit_checklist_criteria%'")
```

Comment:
What's the format of a subscriber_list tag name?

I thought it was: `brexit_checker_criteria-abc1231231` or something?
if so could we drop the first % or is there something before it?

I'm always unnessiarily nevous of fuzzy matching strings that could be in the middle of others.
Matching from a known-start to unknown end... feels a tad safer, if we can get away with it?

Reply:
Ah, that's a slug you're thinking of.  The tags look like:

`{ brexit_checklist_criteria: { any: ["travelling", "nationality-eu"] } }`

It's a good shout though and I'll think on it

Reply:
Have done a better query on the tags now I think